### DISCUSSION TITLE
Question 38: Is the reciprocal relationship reflected on Host mammals data automatically? [RB-INS]

### DISCUSSION THREAD

**Discussion by rondlg** [fieldmuseum/EMu-Documentation#547] [Q&A] [answered]
Description: When a parasite is linked to the host catalog record, is the reciprocal relationship automatically listed on the Relationship tab or should this be edited in on both records? Thanks!

**Comment by rondlg** [comment, marked as answer]
Response: No. They need to be edited in both records.

**Reply by fmjjones** [reply]
Technically the reciprocal relationship can be made in EMu, the issue is not having a standard definition of the vocabulary of relationships possible and their relationship to one another (e.g., part; counter part). Once a standard is defined and agreed upon at TDWG, it would be possible to ask Axiell to modify the logic on the Relationship tab.
